### ISSUE TITLE
Nice to have, create cards in backlog

### ISSUE THREAD

**Issue by ejouvin** [digitalfondue/lavagna#63]
I may missed something, but in order to create cards, it have to be done from a column.

I did not see the option to create the card directly in Backlog. If I want to plan some "releases", I have to create all my cards in a "dummy" column, and then move all them.

So what about having the create card action enable for backlog ?

**Comment by syjer** [comment]
you are right, it's missing :). Should be easy to implement (and we have enough space in the UI)
